src/Telegram.Bot/ITelegramBotClient.cs

```diff
@@ -875,15 +875,47 @@ public interface ITelegramBotClient
         /// <param name="parseMode">Send Markdown or HTML, if you want Telegram apps to show bold, italic, fixed-width text or inline URLs in the media caption.</param>
         /// <param name="replyMarkup">A JSON-serialized object for an inline keyboard.</param>
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
-        /// <returns><c>true</c> on success.</returns>
-        /// <see href="https://core.telegram.org/bots/api#editmessagecaption"/>
+        /// <returns>On success, the edited Description is returned.</returns>
```

Comment:
This is inline version of the method. It return true on success (Task in our case since there is no point in actually returning it).